test/swar/BasicOperations.cpp

```diff
@@ -323,3 +337,76 @@ constexpr auto aBooleansWithTrue = booleans(SWAR<4, u32>{0x1});
 static_assert(aBooleansWithTrue);
 static_assert(!aBooleansWithTrue); // this is a pitfall, but lesser evil?
 static_assert(false == !bool(aBooleansWithTrue));
+
+TEST_CASE(
+    "fullAddition",
+    "[swar]"
+) {
+    {
```

Comment:
Perhaps use a Catch2 `SECTION` to annotate what this does?

In any case, what is happening in this block is the verification that, lane-wise, `INT_MAX + 1 == INT_MIN`, the numeric comment is very welcome, but what about using a textual description too?